templates/openstack/what-is-openstack.html

```diff
@@ -217,7 +612,130 @@ <h2>How to install OpenStack</h2>
+          </div>
+          <h3 class="p-heading--four">
+            <a href="/engage/openstack-deployment-guide" class="p-link--inverted">OpenStack deployment guide</a>
+          </h3>
+        </div>
+      </div>
+      <div class="col-4 p-divider__block">
+        <div class="p-heading-icon--muted">
+          <div class="p-heading-icon__header">
+            <div>
+              {{ image (
+                url="https://assets.ubuntu.com/v1/005c3416-webinar-white.svg",
+                alt="",
+                width="171",
+                height="150",
+                hi_def=True,
+                loading="lazy"
+                ) | safe
+              }}
+            </div>
+            <h4 class="p-heading-icon__title" style="color: #fff;">WEBINAR</h4>
+          </div>
+          <h3 class="p-heading--four">
+            <a href="https://www.brighttalk.com/webcast/6793/387966?utm_source=brighttalk-portal&utm_medium=web&utm_content=hosted%20private%20cloud%20infrastructure&utm_campaign=webcasts-search-results-feed" class="p-link--external p-link--inverted">Hosted private cloud infrastructure: a cost analysis</a>
+          </h3>
+        </div>
+      </div>
+      <div class="col-4 p-divider__block">
+        <div class="p-heading-icon--muted">
+          <div class="p-heading-icon__header">
+            <div>
+              {{ image (
+                url="https://assets.ubuntu.com/v1/30037eac-datasheet-white.svg",
+                alt="",
+                width="171",
+                height="150",
+                hi_def=True,
+                loading="lazy"
+                ) | safe
+              }}
+            </div>
+            <h4 class="p-heading-icon__title" style="color: #fff;">DATASHEET</h4>
+          </div>
+          <h3 class="p-heading--four">
+            <a href="https://docs.google.com/document/d/1LTpZyxrRYofxxMtrH6E4khmEoYJkgs0_RntUdpcV0AI/edit#heading=h.1b0cu17ni6v" class="p-link--external p-link--inverted">Canonical’s Charmed OpenStack</a>
+          </h3>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="u-fixed-width">
+      <h2>All the pieces you need to start using OpenStack</h2>
+  </div>
+  <div class="row u-equal-height">
+    <article class="col-6 p-card">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/a7916513-picto-openstack.svg",
+            alt="",
+            width="50",
+            height="50",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+          <h3 class="col-10 p-card__title p-heading--three">Canonical’s Charmed OpenStack</h3>
+        </div>
+      </div>
+      <hr class="u-sv1" />
+      <p class="p-card__content">Canonical’s Charmed OpenStack is a pure upstream OpenStack distribution engineered for the best price-performance with full enterprise support.</p>
```

Comment:
This sometimes happens when you copy and paste from the copy doc, but the quotes and apostrophes should all be straight 
```suggestion
      <p class="p-card__content">Canonical's Charmed OpenStack is a pure upstream OpenStack distribution engineered for the best price-performance with full enterprise support.</p>
```